crates/shared/src/http_solver/model.rs

```diff
@@ -239,6 +243,24 @@ pub struct ExecutedOrderModel {
     pub exec_plan: Option<ExecutionPlan>,
 }
 
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ExecutedLiquidityOrderModel {
+    pub order: NativeLiquidityOrder,
+    #[serde(with = "u256_decimal")]
+    pub exec_sell_amount: U256,
+    #[serde(with = "u256_decimal")]
+    pub exec_buy_amount: U256,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct NativeLiquidityOrder {
+    pub from: H160,
+    #[serde(flatten)]
+    pub data: OrderData,
+    #[serde(flatten)]
+    pub signature: Signature,
+}
```

Comment:
How do you want to handle partially fillable orders which don't get filled completely? I see that your test creates a partially fillable order so I assume they are allowed.
Intuitively I would think partially fillable orders are not required to solve the problem of PMM integration.
Do you plan to store the just in time created orders in the database eventually?

Reply:
> How do you want to handle partially fillable orders which don't get filled completely?

Currently, there are no special semantics for them. Specifically, the execution amount is part of the `ExecutedLiquidityOrderModel`, so the order will get encoded with however much is intended to be executed with.

> Intuitively I would think partially fillable orders are not required to solve the problem of PMM integration.

They are not - I decided to not disallow them as they didn't really require any additional logic to support (and would require additional logic to deny). Besides, a logical integrator will use fill-or-kill orders for minor gas efficiency.

> Do you plan to store the just in time created orders in the database eventually?

Don't think its needed. If there are left-overs, then "smart" external solvers can keep an eye on the settlements from other solvers and index these orders independently. I don't forsee this being an issue however.

Reply:
>Currently, there are no special semantics for them. Specifically, the execution amount is part of the ExecutedLiquidityOrderModel, so the order will get encoded with however much is intended to be executed with.

I wasn't really concerned about encoding the orders but more with what happens with those partially filled orders. But if we don't really care about them (e.g. we don't store them for later use) then allowing partially fillable orders seems fine.

Reply:
I think they can exist as "throw-away" orders just like their fill-or-kill cousins. I'm not too concern about having left-over partially filled orders TBH.